src/API/Common/Time/ClockInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Common\Time;
+
+interface ClockInterface
+{
+    public const MILLIS_PER_SECOND = 1_000;
+    public const MICROS_PER_SECOND = 1_000_000;
```

Comment:
Do we need these two constants? Only conversion from/to nanos should be relevant.

Reply:
we don't. removed.